docs/source/models/concept_models/vivarium_nutrition_optimization/concept_model.rst

```diff
@@ -57,14 +57,77 @@ Documents that contain information specific to the subcomponents of this simulat
+
+2.1 General modeling strategy
+-----------------------------
+
+2.1.1 Conceptualizing "mixed" intervention coverage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To achieve the general modeling objective of determining the most cost effective combination of interventions for a given budget size. When we think about "*combinations* of interventions," it is important to distinguish between two modes in which coverage of more than one intervention may occur. For example, imagine a scenario in which we have 50% population coverage of intervention A and 50% population coverage of intervention B. Any of the following could be true:
+
+1. Combined intervention coverage among the same individuals: 50% of the population is covered by intervention A and B, 50% of the population is covered by neither intervention A nor B
+
+2. Combined intervention coverage among separate individuals: 50% of the population is covered by intervention A, 50% of the population is covered by intervention B
+
+3. Some combination of 1 and 2
+
+
+**In the case of the first situation:**
+
+  There may be important interactions between the *simultaneous* coverage of intervention A and B in the same subpopulation. For example:
+
+  - An infant who was covered by antenatal supplementation may survive the neonatal period due to associated improvements to their LBWSG exposure and therefore survive to benefit from SQLNS supplementation later in life
+  - A child who may have benefitted from SAM treatment alone may no longer need to be treated for SAM if they are covered by MAM treatment as well as SAM treatment
+  - Etc.
+
+  These interactions may be difficult to quantify directly and therefore we will plan to estimate the effect of joint coverage of multiple interventions on maternal and child health from within our simualtion.
+
+**In the case of the second situation:**
+
+  For our purposes, we assume that the situation described in point #2 above is equivalent to the weighted average of 50% an identical population with 100% coverage of intervention A and 50% an identical population with 100% coverage of intervention B. 
+
+    NOTE: see a discussion of the potential limitations of this assumption as it relates to simulated common random numbers in the `6.0 Limitations`_ section.
+
+  As this assumption requires only a weighted average, we can perform this calculation outside of our simualtion so long as we have quantified the impacts of intervention A and B within our simulation already.
+
+**In the case of the third situation:**
+
+  Then, we can combine our strategies for the first two situations so that we may estimate more complex combinations of joint intervention coverage (for example: 50% of the population covered by A and B, 25% covered by A, 25% covered by neither, for total poulation coverage of 75% for A, 50% for B).
```

Comment:
```suggestion
  Then, we can combine our strategies for the first two situations so that we may estimate more complex combinations of joint intervention coverage (for example: 50% of the population covered by A and B, 25% covered by A, 25% covered by neither, for total population coverage of 75% for A, 50% for B).
```